packages/app-root/src/app/users/[login]/stats/page.js

```diff
@@ -1,9 +1,12 @@
 'use client'
-
 import { UserStats } from '@zooniverse/user'
+import auth from 'panoptes-client/lib/auth'
 
-export default function UserPage() {
+export default function UserPage({ params }) {
   return (
-    <UserStats />
+    <UserStats
+      authClient={auth}
+      login={params.login}
```

Comment:
Why is `login` grabbed from `params` in app-root to be passed to ProfileHeader in lib-user? Doing so results in a scenario where I can put a non-existent `login` into the url with app-root run locally, and UserStats renders my authenticated `user`'s stats, but the login is from the url 🤔 Why not get a volunteer's `login` from `user` via the AuthClient in UserStats? Or is the necessary data not available there?

<img width="800" alt="Screenshot 2024-02-20 at 2 43 00 PM" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/aa35878a-2deb-4460-ae05-c1066820fe81">

Reply:
Yes great note, I'm getting way ahead of myself by using `login`. Eventually, I think we want admin access to a user's stats page in which case I'd have to refactor related hooks and there'd be a request based on url login with admin auth, though I have to double check that I'm not 100% sure off the top of my head, and regardless, it's not necessary now and is confusing. I'm going to refactor to use `user.login` and revisit admin access if/when applicable.